fraud/pb/proof.proto

```diff
@@ -16,3 +16,14 @@ message BadEncoding {
   uint32 Index    = 4;
   axis Axis    = 5;
 }
+
+
+message RespondedProof {
```

Comment:
"Responded" isn't an adjective in English, so this sounds a little off - would `ProofResponse` work?